devTools/svgTester/export-graphs.ts

```diff
@@ -19,7 +19,11 @@ async function main(parsedArgs: parseArgs.ParsedArgs) {
         const directories = []
         for await (const entry of dir) {
             if (entry.isDirectory()) {
-                directories.push(entry.name)
+                if (String(targetConfig) === entry.name) {
```

Comment:
Just curious: what does the String constructor do here?

Reply:
Oop! A leftover from an earlier attempt which used fs to check if targetConfig existed, and threw an error because `parsedArgs` was exporting it as a number.

Switching to a loose equality check 👍